flake.nix

```diff
@@ -147,9 +147,17 @@
         mkJupyterlabInstance = {
           kernels ? k: {}, # k: { python: k.python {}; },
           extensions ? e: [], # e: [ e.jupy-ext ],
+          runtimePackages ? [],
         }: let
           defaultKernelsPath = self + "/kernels";
 
+          allRuntimePackages =
+            runtimePackages
+            ++ (with pkgs; [
```

Comment:
We should probably add a comment here why we added this 2 dependencies. Most likely we would like to put this impure stuff under some flag.